app/models/emoji.rb

```diff
@@ -150,6 +158,25 @@ def self.load_standard
     db["emojis"].map { |e| Emoji.create_from_db_item(e) }.compact
   end
 
+  def self.load_allowed
+    denied_emojis = denied
+
+    if denied_emojis.present?
+      all.reject { |e| denied_emojis.include?(e.name) }
+    else
+      all
```

Comment:
We're kind of double caching here since `Emoji.all` is already cached. This ends up adding more data to our cache unnecessarily.

Reply:
Actually.... this also means that if the number of denied emojis is low, we also end up double caching alot of the data :thinking: 

```
[5] pry(main)> now = Time.zone.now; Emoji.all.reject { |emoji| ["alien", "robot"].include?(emoji.name) }; Time.zone.now - now
=> 0.003174472
```

Looping through all the emojis add 3ms to each page load though :thinking:

Reply:
Yea sorry for the confusion, let's just remove the cache for now.